### PR TITLE
CentOS / RHEL instructions

### PR DESCRIPTION
**Pull request by denfromufa**
Include and lib paths have to be specified explicitly on CentOS / RHEL when LD_LIBRARY_PATH is empty

### CODE REVIEW

**Comment by alsrgv** [comment]
@denfromufa, thanks!  Can you sign the DCO (`git rebase --signoff master` and force push)?

File: docs/gpus.rst [hunk 1]
```diff
@@ -53,6 +53,12 @@ by installing an `nv_peer_memory <https://github.com/Mellanox/nv_peer_memory>`__
    .. code-block:: bash
 
        $ HOROVOD_GPU_ALLREDUCE=NCCL pip install --no-cache-dir horovod
+   
+   If you have installed NCCL 2 using the CentOS / RHEL package, you can run:
```
Comment (sblotner) [inline docs/gpus.rst]: have installed --> installed

Comment (alsrgv) [reply docs/gpus.rst]: Let's align the sentence above with the same.

Comment (denfromufa) [reply docs/gpus.rst]: @alsrgv @sblotner do you suggest to correct the grammar in all 5 places with "present perfect tense" in this document?

Comment (denfromufa) [reply docs/gpus.rst]: done!

**Comment by alsrgv** [comment]
@denfromufa, could you rebase on master? It has some break fixes.

**Comment by alsrgv** [comment]
Will merge since build failures are unrelated.

**Pull request merged**